endpoints/events/event.go

```diff
@@ -217,7 +217,7 @@ func HandleAccountServiceErrors(errs []error) (status int, messages []string) {
 		}
 	}
 
-	return status, messages
+	return
```

Comment:
We can simply do `return` and its understood that named parameters `status` and `messages` are returned.